apps/helloworld/quarkus/dockerbuild_openshift.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker build -f kubefiles/Dockerfile.openshift -t dev.local/burrsutter/quarkus-demo:2.0.0 .
```

Comment:
why this is still pointing to Burr's repo ?

Reply:
I guess these are helpers script for @burrsutter and people won't actually really push to our repo so I did not touched them but I can change it